git-subhistory.sh

```diff
@@ -16,10 +16,11 @@ test $# = 0 && set -- -h
 
 OPTS_SPEC="\
 git-subhistory split <subproj-path> (-b | -B) <subproj-branch>
-git-subhistory merge <subproj-path> <subproj-branch>
+git-subhistory merge <subproj-path> (-a) <subproj-branch>
```

Comment:
this should be square brackets `[]` since it's optional. Also it should be at the end after `<subproj-branch>` which is required. So like:
```
git-subhistory merge <subproj-path> <subproj-branch> [--no-edit]
```